isso/js/app/text/postbox.jade

```diff
@@ -5,13 +5,16 @@ div(class='isso-postbox')
                 = i18n('postbox-text')
         section(class='auth-section')
             p(class='input-wrapper')
-                input(type='text' name='author' placeholder=i18n('postbox-author')
+                label(for='isso-postbox__author',class='isso-visuallyhidden')= i18n('postbox-author')
```

Comment:
Looks good, but can you fix the whitespace around the '=' before i18n? I think adding a space there would be best. *nitpick* :) Same with the two other lines.